### PR TITLE
FFA-36 Avoid seeing login when the user is logged in

### PR DESCRIPTION
**Pull request by rndevelo**

### CODE REVIEW

File: app/src/main/java/com/digitalsolution/familyfilmapp/MainViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            delay(100)
+            _isLoading.value = false
+        }
+    }
+
+    fun hideSplashScreen(){
+//        Log.d("SPLASH", "hideSplashScreen1: ${_isLoading.value}")
+//        _isLoading.value = false
+//        Log.d("SPLASH", "hideSplashScreen2: ${_isLoading.value}")
+    }
```
Comment (hgarciaalberto) [inline app/src/main/java/com/digitalsolution/familyfilmapp/MainViewModel.kt:29]: It it does nothing, it is always better to remove unnecessary code

**Review by hgarciaalberto — CHANGES_REQUESTED** [review]
Perfect implementation but just a few improvements... 
Well done crack!

File: app/src/main/java/com/digitalsolution/familyfilmapp/MainActivity.kt [hunk 2]
```diff
@@ -3,19 +3,25 @@ package com.digitalsolution.familyfilmapp
...
 import dagger.hilt.android.AndroidEntryPoint
 
 @AndroidEntryPoint
 class MainActivity : ComponentActivity() {
+
+    private val viewModel: MainViewModel by viewModels()
+
     override fun onCreate(savedInstanceState: Bundle?) {
+        installSplashScreen().setKeepOnScreenCondition { viewModel.isLoading.value }
         super.onCreate(savedInstanceState)
```
[the remark this answers is not in the document]
Comment (rndevelo) [reply app/src/main/java/com/digitalsolution/familyfilmapp/MainActivity.kt:25]: I think that this should be like this, we can test it.

**Review by hgarciaalberto — APPROVED** [review]
Looks fantastic!! 🚀

**Pull request merged**